proto/spaceone/api/inventory_v2/v1/namespace_group.proto

```diff
@@ -0,0 +1,138 @@
+syntax = "proto3";
+
+package spaceone.api.inventory_v2.v1;
+option go_package = "github.com/cloudforet-io/api/dist/go/spaceone/api/inventory-v2/v1";
+
+
+import "google/protobuf/empty.proto";
+import "google/protobuf/struct.proto";
+import "google/api/annotations.proto";
+import "spaceone/api/core/v2/query.proto";
+
+
+service NamespaceGroup {
+    rpc create (CreateNamespaceGroupRequest) returns (NamespaceGroupInfo) {
+        option (google.api.http) = {
+            post: "/inventory-v2/v1/namespace-group/create"
+            body: "*"
+        };
+    }
+
+    rpc update (UpdateNamespaceGroupRequest) returns (NamespaceGroupInfo) {
+        option (google.api.http) = {
+            post: "/inventory-v2/v1/namespace-group/update"
+            body: "*"
+        };
+    }
+
+    rpc delete (NamespaceGroupRequest) returns (google.protobuf.Empty) {
+        option (google.api.http) = {
+            post: "/inventory-v2/v1/namespace-group/delete"
+            body: "*"
+        };
+    }
+
+    rpc get (NamespaceGroupRequest) returns (NamespaceGroupInfo) {
+        option (google.api.http) = {
+            post: "/inventory-v2/v1/namespace-group/get"
+            body: "*"
+        };
+    }
+
+    rpc list (NamespaceGroupQuery) returns (NamespaceGroupsInfo) {
+        option (google.api.http) = {
+            post: "/inventory-v2/v1/namespace-group/list"
+            body: "*"
+        };
+    }
+
+    rpc stat (NamespaceGroupStatQuery) returns (google.protobuf.Struct) {
+        option (google.api.http) = {
+            post: "/inventory-v2/v1/namespace-group/stat"
+            body: "*"
+        };
+    }
+}
+
+//{
+//
+//}
+message CreateNamespaceGroupRequest {
+    // +optional
+    string namespace_group_id = 1;
+    string name = 2;
+    string icon = 3;
+    // +optional
+    string description = 4;
+    // +optional
+    google.protobuf.Struct tags = 5;
+    string resource_group = 20;
+    // +optional
+    string workspace_id = 21;
+}
+
+//{
+//
+//}
+message UpdateNamespaceGroupRequest {
+    string namespace_group_id = 1;
+    // +optional
+    string name = 2;
+    // +optional
+    string icon = 3;
+    // +optional
+    string description = 4;
+    // +optional
+    google.protobuf.Struct tags = 5;
+}
+
+//{
+//
+//}
+message NamespaceGroupRequest {
+    string namespace_group_id = 1;
+}
+
+//{
+//
+//}
+message NamespaceGroupQuery {
+    // +optional
+    spaceone.api.core.v2.Query query = 1;
+    // +optional
+    string namespace_group_id = 2;
+    // +optional
+    bool exist_only = 3;
+    // +optional
+    string workspace_id = 21;
+}
+
+//{
+//
+//}
+message NamespaceGroupInfo {
+    string namespace_group_id = 1;
+    string name = 2;
+    string icon = 3;
+    // +optional
+    string description = 4;
+    // +optional
+    google.protobuf.Struct tags = 5;
+    bool is_managed= 7;
```

Comment:
It would be better if the numbering for this field starts from 6.

Reply:
ok Update completed